index.html

```diff
@@ -0,0 +1,350 @@
+<!doctype html>
+<html class="no-js" lang="">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>Avengers</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <link rel="apple-touch-icon" href="apple-touch-icon.png">
+
+    <link rel="stylesheet" href="/css/reset.css">
+    <link rel="stylesheet" href="/css/style.css">
+  </head>
+  <body>
+    <nav class="menu">
+      <a href="#home">Home.</a>
+      <a href="#services">Services.</a>
+      <a href="#portfolio">Portfolio.</a>
+      <a href="#about">About.</a>
+      <a href="#team">Team.</a>
+      <a href="#contact">Contact.</a>
+    </nav>
+    <header id="home" class="header-content">
+      <img class="logo circle" width="21" height="32" src="/img/tear-drop-two.svg" alt="Logo for Avengers" title="Avengers logo">
+      <h1 class="header-logo-text">Hi there! We're Avengers, <strong>creative design agency</strong> we build <strong class="curve-underline">awesome</strong> mobile and websites</h1>
+    </header>
+    <main>
+      <article id="services" class="services-container">
```

Comment:
Actually, sorry to reverse position on this -- your use of `article` is appropriate (I got turned around). These can stay as-is, but I'd still recommend fixing the heading rankings within the articles.